From 2561eb22cdbf3e027832f11959b991bcea47c4c0 Mon Sep 17 00:00:00 2001
From: Alejandro Martinez Ruiz <alex@nowcomputing.com>
Date: Thu, 24 Jan 2013 20:56:25 +0100
Subject: [PATCH] Revert "f2fs: add remap_pages as generic_file_remap_pages"

This reverts upstream commit 692bb55d1ab5b278181ff2e65f09eb0be6d50669.

The original commit was meant to include support for a feature not
present in 3.5 kernels.

Signed-off-by: Alejandro Martinez Ruiz <alex@nowcomputing.com>
---
 fs/f2fs/file.c | 5 ++---
 1 file changed, 2 insertions(+), 3 deletions(-)

diff --git a/fs/f2fs/file.c b/fs/f2fs/file.c
index 24b8cda..207b180 100644
--- a/fs/f2fs/file.c
+++ b/fs/f2fs/file.c
@@ -105,9 +105,8 @@ out:
 }
 
 static const struct vm_operations_struct f2fs_file_vm_ops = {
-	.fault		= filemap_fault,
-	.page_mkwrite	= f2fs_vm_page_mkwrite,
-	.remap_pages	= generic_file_remap_pages,
+	.fault        = filemap_fault,
+	.page_mkwrite = f2fs_vm_page_mkwrite,
 };
 
 int f2fs_sync_file(struct file *file, loff_t start, loff_t end, int datasync)
-- 
1.8.2.1.422.gdcd8c09

